client/app/bundles/course/assessment/submission/submission.jsx

```diff
@@ -1,10 +1,8 @@
 /* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
 
 import { render } from 'react-dom';
-import { Router } from 'react-router';
-import history from 'lib/history';
```

Comment:
We can perhaps delete `lib/history` if this file is no longer used. + maybe uninstall the `history` dep if not utilised.

Reply:
We can keep it as of now. It is used somewhere else in the codebase thats not related to React Router.